src/renderer/tvgRender.h

```diff
@@ -438,6 +439,17 @@ struct RenderEffectGaussianBlur : RenderEffect
         inst->type = SceneEffect::GaussianBlur;
         return inst;
     }
+
+    RenderEffectGaussianBlur* duplicate() const override
+    {
+        auto inst = new RenderEffectGaussianBlur;
```

Comment:
There’s no need to write code unnecessarily verbosely. One-line assignment is enough for copy structure data...